plans/provision/virtual.fmf

```diff
@@ -45,3 +45,12 @@ adjust+:
                 is-supported: true
             memory: ">= 4 GB"
```

Comment:
```suggestion
            memory: ">= 8 GB"
```

What about raising the memory a bit as well?

Reply:
I would rather not change two parameters at once. 1. the machine was barely working, and 2. if we change two properties, it would be hard to tell which of them, if any, affected the speed of testing and how much.

Reply:
Ok, makes sense.